Makefile.toml

```diff
@@ -246,6 +254,24 @@ args = [
 ]
 dependencies = ["litcheck"]
 
+[tasks.test-rust-cov]
+category = "Test"
+description = "Runs tests written in Rust and generates code coverage report"
+command = "cargo"
+args = [
+    "tarpaulin",
+    "--timeout=360",
+    "--out",
+    "lcov",
+]
+dependencies = [
+    "install-wasm-target",
```

Comment:
Now that we use `rust-toolchain.toml`, it occurs to me that we don't need to explicitly install the targets/components anymore, we just need to make sure they are in the toolchain file.